graphql_api/types/0_TEMP/coverage.graphql

```diff
@@ -0,0 +1,51 @@
+# Coverage is information related to a repo's
+# main branch test coverage
+type CoverageAnalysis {
+  hits: Int
+  misses: Int
+  lines: Int
+  coverageSha: String
+  percentCovered: Float # formerly repo.coverage
```

Comment:
We have the option of using the `CoverageTotals` [type](https://github.com/codecov/codecov-api/blob/324fd728cd905c073e96c6657ac3d363a92c6070/graphql_api/types/coverage_totals/coverage_totals.graphql#L2) instead of these flat fields `hits`, `mises`, etc.